app/controllers/experiments_controller.rb

```diff
@@ -517,7 +520,7 @@ def view_type_params
   def check_read_permissions
     current_team_switch(@experiment.project.team) if current_team != @experiment.project.team
     render_403 unless can_read_experiment?(@experiment) ||
-                      @experiment.archived? && can_read_archived_experiment?(@experiment)
+                      (@experiment.archived? && can_read_archived_experiment?(@experiment))
```

Comment:
was this change intentional?

Reply:
No, I think rubocop recommended it!
Should I change it back to how it was?